### ISSUE TITLE
Suggest to replace redundant wildcard patterns with `..`

### ISSUE THREAD

**Issue by zeroeightysix** [intellij-rust/intellij-rust#6409]
Consider any enum with variants with more than one item:
```rust
enum Enum {
    A(usize, usize),
    B(String, usize, bool)
}
```

```rust
let e = Enum::A(1, 2);

match e {
    Enum::A(_, _) => { // Can be replaced with Enum::A(..) =>
        println!("{}", "hello world!");
    }
    _ => ()
};
```

```rust
if let Enum::B(a, _, _) = e { // Can be replaced with Enum:B(a, ..) =>
    println!("{}, {}!", "hello", a)
}
```

It might be useful to provide an intent (maybe with even a warning, such as the unnecessary parentheses warning) to convert redundant usages of multiple `_` to one `..`.
